packages/playground/src/components/logger.vue

```diff
@@ -203,7 +203,7 @@ export default {
     let _interceptorQueue: LI[] = [];
 
     async function interceptMessage(instance: LI) {
-      if (connectDB.value.error) {
+      if (connectDB?.value?.error) {
```

Comment:
there are still some errors coming from the logger 
https://github.com/threefoldtech/tfgrid-sdk-ts/blob/431d55a9b56dff5a2ffc4011f0da6bb704a465b0/packages/playground/src/components/logger.vue#L215
![image](https://github.com/user-attachments/assets/02ea68de-17e7-4885-8bfc-f83d0f9bb9b9)

Reply:
this issue caused due to nullable prototype of object

![image](https://github.com/user-attachments/assets/5a36f992-f951-4f98-aff2-3bd64f9ab400)